packages/pigeon/bin/pigeon.dart

```diff
@@ -33,8 +33,8 @@ Future<void> main(List<String> args) async {
     final String relInputPath = _posixRelative(opts.input, from: tempDir.path);
     importLine = 'import \'$relInputPath\';\n';
   }
-  final String code = """$importLine
-// @dart = 2.2
+  final String code = """// @dart = 2.2
```

Comment:
is 2.2 right?

Reply:
Yea, that's the minimum version that is in the specified in the pigeon pubspec.yaml.